src/main.rs

```diff
@@ -138,6 +160,17 @@ fn main() {
                 .short("i")
                 .help("Ignore non-zero exit codes."),
         )
+        .arg(
+            Arg::with_name("parameter-range")
```

Comment:
If we're always expecting two values we may want to defer to clap to ensure they are both there by using [```number_of_values(2)```](https://docs.rs/clap/2.31.2/clap/struct.Arg.html#method.number_of_values)

Reply:
I added this in the beginning, but it doesn't seem to be necessary because we use `.value_names(&["VAR", "MIN", "MAX"])`, which seems to set this as well.